### DISCUSSION TITLE
privacy for Android

### DISCUSSION THREAD

**Discussion by nayfalgrba** [community/community#42103] [Mobile]
I want best download for my phone in Google play

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @nayfalgrba! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

Also, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Sxsky81** [reply]
- [ ] - **_[](https://tx.botdev.me/TRX/831697d38822894842c662c4d784da2aa48688ac3a34502629873c32e778c8ae)_**@
